common/app/views/fragments/atoms/interactive.scala.html

```diff
@@ -0,0 +1,44 @@
+@(interactive: _root_.model.content.InteractiveAtom, shouldFence: Boolean)
+
+@body = {
+    <style>
+        @HtmlFormat.raw(interactive.css)
+    </style>
+    @HtmlFormat.raw(interactive.html)
+}
+
+@iframeBody = {
+    @body
+    <script>
+        function resize() {
+            window.frameElement.height = document.body.offsetHeight;
+        }
+
+        var fonts = [].slice.apply(window.parent.document.styleSheets)
+            .filter(function (sheet) { return sheet.ownerNode.className.indexOf("webfont") > - 1; })
+            .map(function (sheet) { return sheet.ownerNode.textContent; })
+            .join(' ');
+        var css = document.createElement('style');
+        css.textContent = fonts;
+        document.head.appendChild(css);
```

Comment:
Before anyone says it, this is obviously not very nice. But I'm interested to talk about how we can get fonts loaded into the iframes fast.